product/index.md

```diff
@@ -2,6 +2,17 @@
 
 This covers how we run our business and deliver service to our customers.
 
+## Product Vision
+Our target customers are users familiar with Node-RED looking to run multiple instances or share access across multiple users.
+
+We make it easy to create instances of Node-RED and to share configuration and work between them. We provide collaboration functionality allowing multiple people in a team to build and maintain a flow. 
+We give user a management platform that delivers the features around running node-red such that it requires minimal effort on the part of the customer to configure, update, and maintain their environments.
```

Comment:
What FlowForge already provides and what we are building towards isn't differentiate enough I think for people unfamiliar with our product. Maybe it should describe what we offer now, and where we're building towards?

Also: s/node-red/Node-RED/

Reply:
As a vision though it should be forward looking rather than a product spec of what is currently offered, details of the current product are better suited to a product page on the website